app.js

```diff
@@ -88,8 +116,14 @@ function refreshCurrentGameEl() {
 
 function displayAllGames() {
     // clear out the past games list in the DOM
-
+    pastGamesEl.textContent = '';
     // loop through the past games in state
+    for (let game of pastGames){
+        const gameEl = renderGame(game.name1, game.name2, game.score1, game.score2);
+        pastGamesEl.append(gameEl);}
     // use the renderGame function to render and append a past game for each past game in state
+    renderGame();
```

Comment:
the renderGame on 125 is also an unnecessary function call since you're already doing it on 122 and appending on 123